asset_dashboard/views.py

```diff
@@ -12,13 +13,34 @@
 
 class CipPlannerView(TemplateView):
     title = 'CIP Planner'
-    template_name = 'asset_dashboard/index.html'
-    component = 'js/pages/index.js'
+    template_name = 'asset_dashboard/planner.html'
+    component = 'js/planner.js'
 
     def get_context_data(self, *args, **kwargs):
         context = super().get_context_data(*args, **kwargs)
-        projects = serializers.serialize('json', Project.objects.all())
-        context['props'] = {'projects': projects}
+
+        projects = Project.objects.all()
+        projects = projects.select_related('section_owner', 'category', 'projectfinances', 'projectscore')
+
+        projects_list = []
+        for prj in list(projects):
```

Comment:
i don't think you need to cast to a list here.

Reply:
Fixed here: 6add908